src/Umbraco.Web/UmbracoDbProviderFactoryCreator.cs

```diff
@@ -40,5 +42,11 @@ public ISqlSyntaxProvider GetSqlSyntaxProvider(string providerName)
                     throw new InvalidOperationException($"Unknown provider name \"{providerName}\"");
             }
         }
+
+        public void CreateDatabase()
+        {
+            var engine = new SqlCeEngine(DatabaseBuilder.EmbeddedDatabaseConnectionString);
```

Comment:
This seems odd since we aren't checking for a provider type here or anything, are we checking what is specified before we call this?